crates/floresta-chain/src/pruned_utreexo/consensus.rs

```diff
@@ -102,6 +100,83 @@ impl Consensus {
         sha256::Hash::from_slice(leaf_hash.as_slice())
             .expect("parent_hash: Engines shouldn't be Err")
     }
+
+    /// Check whether a block is valid
+    ///
+    /// This function is divided into two parts:
+    ///
+    /// Header validation with [`Consensus::validate_header`]
+    ///
+    /// Transaction validation with [`Consensus::validate_block_transactions`]
+    pub fn validate_block(
+        &self,
+        block: &bitcoin::Block,
+        ancestor: &BlockHeader,
+        height: u32,
+        utxo_data: UtxoMap,
+        flags: c_uint,
+        validate_script: bool,
+        _node_time: &impl NodeTime,
```

Comment:
Why is this `_node_time` here? We won't need it in this function

Reply:
good eye, its was set before we decided to split MTP and node_time...